cmd/ci-scheduling-webhook/mutation.go

```diff
@@ -301,7 +302,17 @@ func mutatePod(w http.ResponseWriter, r *http.Request) {
 			klog.Errorf("No node precludes will be set in pod due to error: %v", err)
 		}
 
+		highPerfPod := false
 		if podClass == PodClassBuilds {
+			// Use high performance nodes for large pods
+			memoryThreshold := resource.MustParse("32Gi")
```

Comment:
You could move `memoryThreshold` and `cpuThreshold` to the package variable definitions section at the top of this file since their value is fixed.